collection_functions.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+
+function getCampaignNames($db)
+{
+    $campaigns = $db->prepare("SELECT DISTINCT `cycle` FROM `scenarios`;");
+    $campaigns->setFetchMode(PDO::FETCH_ASSOC);
+    $campaigns->execute();
+    $cycleArray = array();
+    $results = $campaigns->fetchAll();
+    foreach ($results as $cycleName) {
+        array_push($cycleArray, $cycleName["cycle"]);
+    }
+    return $cycleArray;
+}
+
+function getScenarios($db)
+{
+    $queryString = "SELECT * FROM `scenarios` ORDER BY `position`;";
+    $query = $db->prepare($queryString);
+    $query->setFetchMode(PDO::FETCH_ASSOC);
+    $query->execute();
+    $bigArray = $query->fetchAll();
```

Comment:
variable naming - might not always be big something like $scenarios is probably more suited

Reply:
now call $allScenarios throughout the code